plugins/inputs/cadvisor/cadvisor.go

```diff
@@ -1,8 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT
 
-//go:build linux || darwin
-// +build linux darwin
+//go:build linux
```

Comment:
For cadvisor plugin, would we consider adding build constraint for windows?

Reply:
Cadvisor is for linux only. If we want to support windows we would have to use something else.